libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -934,16 +934,21 @@ void ofFbo::updateTexture(int attachmentPoint) {
 			glPushAttrib(GL_COLOR_BUFFER_BIT);
 		}
 
-		bind();
+		// Because we temporarily change global Fbo binding state
+		// when blitting from one framebuffer to the other, we first have 
+		// to query the renderer for its binding state, and store it.
+		GLuint rendererFboBindingState = ofGetGLRenderer()->getCurrentFramebufferId();
```

Comment:
this isn't already in ofGLRenderer, no?  in any case this call to a singleton makes any dependent call thread unsafe. i can take a look later to see if there's any way to do this without using the singleton